src/workerd/io/compatibility-date.capnp

```diff
@@ -732,4 +732,11 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
       $compatDisableFlag("allow_importable_env");
   # When allowed, `import { env } from 'cloudflare:workers'` will provide access
   # to the per-request environment/bindings.
+
+  abortSignalNotIgnoredForSubrequests @79 :Bool
+      $compatEnableFlag("abort_signal_not_ignored_for_subrequests")
+      $compatDisableFlag("abort_signal_ignored_for_subrequests");
```

Comment:
I'm assuming it is intentional that this has no default on date? If so, might be good to expand the doc comment to indicate so, but good without that also.

Reply:
I was copying your "disableImportableEnv" -- I'm still a bit shaky on compat flags.

 I'd say the reasoning is that we won't enable "abortSignalNotIgnoredForSubrequests" because it's potentially not backwards compatible, and we haven't yet seen a clear use case that needs "abortSignalNotIgnoredForSubrequests". Does that make sense to you?

Reply:
I think this is exactly the sort of change where we want a default-enable date. 99.99% of people won't be affected and the new behavior is "better".

Setting a default-enable date doesn't impact existing workers -- they have to actually update their compatibility date to be past the default-enable date before it takes effect.

Of course, we need to make sure all flags with default-on dates are documented here, so people can review what's changing when they update their compat date: https://developers.cloudflare.com/workers/configuration/compatibility-flags/

Reply:
How do I decide the default enable date - is it just today, then?